Submission arpril 1st (#617)
ClaudeMetz/Alt-F4: submissions/april-1st/submission-april-1st.md

# April 1st Mod Feature

(v 1.5)

## Working title: Funny Factorio Fan Formulations

> On the special occasion of Friday April 1st, we are proud to present some humorous picks from Factorio's vast modding scene. We will start with mods that only change visuals or audio to add some unique flavors to your eningeering experiences. Afterwards, we look at some crazy new weapons that spice up combat. Finally, we feature a mod with a level of transportation pizzaz that Willy Wonka himself would be proud of.

Reviewer: ```suggestion
On the special occasion of Friday April 1st, we are proud to present some humorous picks from Factorio's vast modding scene. We will start with mods that only change visuals or audio to add some unique flavors to your eningeering experiences. Afterwards, we look at some crazy new weapons that spice up combat. Finally, we feature a mod with a level of transportation pizzazz that Willy Wonka himself would be proud of.
```

### Sights and Sounds

> The mod portal features quite a few mods small and large that offer changes to visuals and/or audio. Here is a fun selection:

Reviewer: ```suggestion
The mod portal features quite a few mods, small and large, that offer visual and/or auditory changes. Here is a fun selection:
```

**[Emotes and Speech Bubbles](https://mods.factorio.com/mod/EmotesAndSpeechBubbles)** 	*Kiplacon*

> This handy mod makes multiplayer interactions a lot livelier, especially when you can't use a mic. Your text in chat also appears over your head. This is already more immersive than following the console output but the even better feature is the ability to use emotes! 

Reviewer: ```suggestion
This handy mod makes multiplayer interactions a lot livelier, especially when you can't use a microphone. Messages you send in chat and emotes you use will appear over your head and will be visible to other players! 
```

> The emotes are displayed by clicking the middle mouse button to open the emote hotbar and left-clicking one to show it. Right-clicking an emote on the hotbar lets you switch it with another from the full library. The mod helps enhance your fun and funny moments: You can point left/right at directions to explore or places of interest. You can vibe with emotes for partying. You can display when you are thinking and boast or praise big-brain moments with the matching emote. And of course, you can "press F" or declare "rekt!" when your friend gets hit by a train! 

Reviewer: ```suggestion
The emotes are displayed by clicking the middle mouse button to open the emote hotbar and left-clicking one to show it. Right-clicking an emote on the hotbar lets you switch it with another from the full library. The mod helps enhance your fun and funny moments: You can point left/right at directions to explore or places of interest. You can vibe with emotes for partying. You can display when you are thinking and boast or praise big-brain moments with the matching emote. And, of course, you can "press F" or declare "rekt!" when your friend gets hit by a train! 
```

*<Video: Kiplacon: Emotes.mp4>*

[**Disco Science**](https://mods.factorio.com/mod/DiscoScience) 	*danielbrauer*

Science is the heart of progress, but via this little mod, it is also the life of the party! It makes research labs light up with the colors of whichever science packs they are consuming. Excellent for sprucing up your [AFK Zone Discos](https://www.reddit.com/r/factorio/comments/sl4z1v/time_honored_tradition_when_a_player_is_afk_you/)! If you want to go further and add faint music to your disco domes, there is [this other mod](https://mods.factorio.com/mod/LabDansen/metrics).

*<Video: disco science>  (not done)*

[**Shrek**](https://mods.factorio.com/mod/shrek) 	*jerzygorski*

> He's back and he means business! This mod retextures your character as Shrek, and proves that he can do rocket science just like how he can rescue the princess! He mines with his fists and he has his own armor and weapon texture too, perfect for getting those pesky bugs out of your swamp planet!

Reviewer: ```suggestion
This mod retextures your character as Shrek, and shows that against all norms and expectations he can do rocket science, just like how he could rescue the princess! He mines with his fists and has his own armor and weapon texture too, perfect for getting those pesky bugs out of your swamp!
```

*<video: shrek mining (not done> (not done)*

*<video: shrek in armor + shooting> (not done)*

[**Bumble Bots**](https://mods.factorio.com/mod/bumble-bots) 	*Maxreader*

> If you ever thought about your bots doing everything for you as your little worker bees, here is the mod that makes it literal! The mod retextures and resounds combat robots, construction and logistic robots, roboports, and logistic chests. The organic structures (especially flowers) among the machines makes an interesing environment for your busy bees to buzz.

Reviewer: ```suggestion
If you ever thought of your bots as your little worker bees, here is a mod making that literal! The mod replaces the textures and sounds of combat robots, construction and logistic robots, roboports, and logistic chests. The organic structures (especially flowers) among the machines make for an interesting environment for your busy bees to buzz in.
```

*<video: logistic bees> (we can reuse part of an older video)*

*<video: defender wasps> (not done, maybe skip)*

[**Sushi Mod**](https://mods.factorio.com/mod/sushi) 	*Xaetral*

> Sushi belt fans, this mod gives you the means to truly flex your skills! It also refreshing for those tired of eating only raw fish! The menu has 6 sushi and 6 maki types. Each can be crafted by one raw fish and has the same properties as it, apart from the enhanced taste. Bon appetit!

Reviewer: ```suggestion
Sushi belt fans, this mod gives you the means to truly flex your skills! It also refreshing for those tired of eating only raw fish! The menu has 6 sushi and 6 maki types. Each can be crafted from one raw fish and has the same properties as it, apart from the enhanced taste. Bon appetit!
```

*<video: sushi on a belt> (not done)*

*OR*

*<image: from mod page>*

[**1000 Sounds to die for**](https://mods.factorio.com/mod/1000sounds) 	*vralfy*

> This sound mod allows you to customize a number of sounds, in particular character deaths. Yes, death in Factorio is usually a tragedy, but you can make it more comedic! Got hit by a train? Hear it honk! Killed by another player? Why not set that to the classic [Wilhelm Scream](https://www.youtube.com/watch?v=r6JK-gRELI0&ab_channel=GamingSoundFX)? You can also customize sounds for a few other events like starting/completing a research, launching a rocket, or players entering/exiting a game. The sound library of this mod borrows from Doom and Quake among some other places. If you like this mod, along the same lines is one for [pain sounds](https://mods.factorio.com/mod/pain) selected from Doom or custom sounds of your choice.

Reviewer: ```suggestion
This sound mod allows you to customize a number of sounds, in particular character deaths. Yes, death in Factorio is usually a tragedy, but you can make it more comedic! Got hit by a train? Hear it honk! Killed by another player? Why not set that to the classic [Wilhelm Scream](https://www.youtube.com/watch?v=r6JK-gRELI0&ab_channel=GamingSoundFX)? You can also customize sounds for a few other events like starting/completing research, launching a rocket, or players entering/exiting a game. The sound library of this mod borrows from Doom and Quake, among some other places. If you like this mod, along the same lines is one for [pain sounds](https://mods.factorio.com/mod/pain) which are selected from Doom or your custom sound pack.
```

*<image: screenshot of mod options in 1.1> (not done)*

[**Choleric cars**](https://mods.factorio.com/mod/cholericcars) 	*Therenas*

> Did your car/tank just crash into rock? Yeah it sucks. This sound mod lets the engineer express his opinion about it very clearly (sometimes with expletives). It definitely captures the feeling of the moment, often comedically well!

Reviewer: ```suggestion
Did your car/tank just crash into a rock again? Yeah, that sucks. This sound mod lets the engineer express his opinion about it very clearly (sometimes with expletives). It definitely captures the feeling of the moment, often comedically well!
```

*<image: crashed car ? > (not done)*

**More sound mods**

There are many more little sound mods out there. For example, Command & Conquer fans might like this pack of [Red Alert sounds](https://mods.factorio.com/mod/Command_and_Conquer_RedAlert_Sounds) for programmable speakers, while [another mod](https://mods.factorio.com/mod/Rick_Roll_mod) lets you even [do this](https://youtu.be/dQw4w9WgXcQ) with speakers. Among the more specific mods, there is one [playing the Thomas the Tank Engine theme](https://mods.factorio.com/mod/thomas-the-tank-engine) when you get hit by a train, while viewers of Trupen might enjoy a custom [rock mining sound](https://mods.factorio.com/mod/TrupenRocks).

### Crazy Combat

> You have many options for combat in vanilla Factorio but there are also many weapon mods that offer crazy new possibilities. And some of these mods truly redefine combat like never before:

Reviewer: ```suggestion
You have many options for combat in vanilla Factorio, but there are also many mods that offer crazy weapons which redefine combat:
```

[**MLG FactOreo**](https://mods.factorio.com/mod/MLG_FactOreo) 	*Kiplacon*

> If combat in Factorio is too bland for you these days and you want to summon the essence of Major League Gaming montages, with all their dubstep-rich glory and dank memes (and some expletives) look no further. This mod delivers. You can craft a sniper rifle for your 360-no-scopepwnage of them noobs. And don't forget to craft meme magazines to deal some aptly named "[dank damage](https://youtu.be/dQw4w9WgXcQ)" with any bullet-firing weapon, including turrets and cars. 

Reviewer: ```suggestion
If combat in Factorio is too bland for you these days and you want to summon the essence of Major League Gaming montages, with all their dubstep-rich glory and dank memes (and some expletives), look no further. This mod delivers. You can craft a sniper rifle for your 360 no-scope pwnage of them noobs. And don't forget to craft meme magazines to deal some aptly named "[dank damage](https://youtu.be/dQw4w9WgXcQ)" with any bullet-firing weapon, including turrets and cars. 
```

*<video: Kiplacon: MLG.mp4>*

[**Train Gun**](https://mods.factorio.com/mod/traingun) 	*Twu.4.25*

> Alright, perhaps you are more interested in a weapon that is more... Factorio-style. We all know that the ultimate killer in the game is no mere biter, but the mighty train! Armed with this knowledge, this mod delivers the formidable lategame superweapon that is the train gun. Unlocked after nuclear missiles, this powerful gun launches nuclear-fueled locomotives that furiously spin and tear through hoardes of enemies and rows of machines alike, before exploding magnificently! Choo choo!

Reviewer: ```suggestion
Alright, perhaps you are more interested in a weapon that is more... Factorio-style. We all know that the ultimate killer in the game is no mere biter, but the mighty train! Armed with this knowledge, this mod delivers the formidable late-game superweapon that is the train gun. Unlocked after nuclear missiles, this powerful gun launches nuclear-fueled locomotives that furiously spin and tear through hoards of enemies and rows of machines alike, before exploding magnificently! Choo choo!
```

*<video: Firing the gun on enemies> (not done)*

### Wonka Level Logistics: 

[**Renai Transportation**](https://mods.factorio.com/mod/RenaiTransportation) 	*Kiplacon*

> And for last on this mod list we have saved the largest and most imaginative. This amazing mod introduces a whole **world** of new transportation options that can liven up your factory by making just about everything soar through the air! While the mod is chaotic, the additions can also be highly practical and you can make cool bases with them, like '[The Dolphin base](https://www.reddit.com/r/factorio/comments/mhrj7v/the_dolphin_base_renai_transportation_one_rail/)' made by famed creator of [JOSEF](https://alt-f4.blog/ALTF4-39/), or this smelting setup that is described by its designer as "[Air speghetti](https://www.youtube.com/watch?v=vZOnYDQOXKU&ab_channel=Shrimp%26EelCo-OpGaming)".

Reviewer: ```suggestion
And for last on this mod list we have saved the largest and most imaginative. This amazing mod introduces a whole **world** of new transportation options that can liven up your factory by making just about everything soar through the air! While the mod is chaotic, the additions can also be highly practical and you can make cool bases with them, like '[The Dolphin base](https://www.reddit.com/r/factorio/comments/mhrj7v/the_dolphin_base_renai_transportation_one_rail/)' made by famed creator of [JOSEF](https://alt-f4.blog/ALTF4-39/), or this smelting setup that is described by its designer as "[Air spaghetti](https://www.youtube.com/watch?v=vZOnYDQOXKU&ab_channel=Shrimp%26EelCo-OpGaming)".
```

Let's look at the mod's features individually:

**Flying items!**

> Renai Transportation makes you ask: Why deal with bots or belts when you can just let the inserters throw the items where they need to go? It introduces thrower inserters that can toss items with precion into open chests that are ready to receive them. You can also ""~~blow~~ install openings onto things" via hatches so that you have direct insertion to machines from a distance!

Reviewer: ```suggestion
Renai Transportation makes you ask: Why deal with bots or belts when you can just let the inserters throw the items where they need to go? It introduces thrower inserters that can toss items with precion into open chests that are ready to receive them. You can also "~~blow~~ install openings onto things" via hatches so that you have direct insertion to machines from a distance!
```

*<video: Kiplacon: Throwing.mp4>*

**Flying you!**

> The mod doesn't stop at items, featuring also player launchers, as a new means of fast travel. Watch where you land, though!

Reviewer: ```suggestion
The mod doesn't stop at items, also featuring player launchers as a new means of fast travel. Watch where you land, though!
```

*<video: Kiplacon: player jump.mp4>*

**Bouncing too!**

> The flight doesn't have to end with one toss. The mod also has bouncepads of all kinds: For both players and items they can extend jumps or change the bounce direction. Some bouncepads can send circuit signals or be set to distribute items across areas. Some items can also be primed on a special bouncepad, allowing things like inserter-thrown grenades or defender capsules!

Reviewer: ```suggestion
The flight doesn't have to end with one toss, as the mod has all kinds of bounce pads: For both players and items they can extend jumps or change the bounce direction. Some bounce pads can send circuit signals or be set to distribute items across areas. Some items can also be primed on a special bounce pad, allowing things like inserter-thrown grenades or defender capsules!
```

*<video: Kiplacon: bounces.mp4>*

**Ride the lightning!**

> If flying isn't your thing, you can instead take a cable car that you can run on your power lines! It lets you zip through the factory without collisions, drawing power from the electric network it uses.

Reviewer: ```suggestion
If being flung across your factory isn't your thing, you can instead take a zipline trolley that runs along your power lines, zooming above your factory!
```

*<video: Kiplacon: zipline.mp4>*

**Flying trains!**

> The mod gets even your trains airborn using ramps and has bouncepads for them too! Admitedly, it is a little hard to precisely set up the ramps but the electrically powered magnetic ramps offer a stable alternative. Factorio doesn't need bridges or train tunnels after all!

Reviewer: ```suggestion
The mod even gets your trains airborne using ramps and bounce pads! Admittedly it is a little hard to precisely set up the ramps, but the electrically powered magnetic ramps offer a stable alternative. Factorio doesn't need bridges or train tunnels after all!
```

*<video: Kiplacon: train.mp4>*

**Instant train unloading!**

> The mod has also found the trick to the fastest way of emptying your trains: Slam them straight into a concrete "unloader" block and let the items just all fly out from specially designed wagons... genius!

Reviewer: ```suggestion
The mod has also found the fastest way of emptying your trains: Slam them straight into a concrete "unloader" block and let the items fly directly onto the belts... genius!
```

*<video: Kiplacon: impact.mp4>*

### And more mods!

> Today we provided an overview of many of the humorous mods on the mod portal, but there more out there and new ones emerging as time goes on. You may ask "how did these come to be?" but we barely scratched the surface. The possibilities are endless when it comes to modding, offering you both the coolest ideas, and the weirdest, as demonstrated by the aptly named modpack [Whyyy](https://mods.factorio.com/mod/Whyyy-modpack/downloads). Many thanks to our modding community who have (and continue to) come up with such colorful additions to Factorio! 

Reviewer: ```suggestion
Today we provided an overview of many of the humorous mods on the mod portal, but there are more out there and new ones are emerging as time goes on. You may ask "How did these come to be?", but we barely scratched the surface. The possibilities are endless when it comes to modding, offering you both the coolest ideas, and the weirdest, as demonstrated by the aptly named modpack [Whyyy](https://mods.factorio.com/mod/Whyyy-modpack/downloads). Many thanks to our modding community who have (and continue to) come up with such colorful additions to Factorio! 
```

### Contributing

> As always, we’re looking for people that want to contribute to Alt-F4. be it by submitting an article or by helping with translation. If you have something interesting in mind that you want to share with the community in a polished way, this is the place to do it. If you’re not too sure about it we’ll gladly help by discussing content ideas and structure questions. If that sounds like something that’s up your alley, join the [Discord](https://alt-f4.blog/discord) to get started!

Reviewer: ```suggestion
As always, we’re looking for people that want to contribute to Alt-F4, be it by submitting an article or by helping with translation. If you have something interesting in mind that you want to share with the community in a polished way, this is the place to do it. If you’re not too sure about it we’ll gladly help by discussing content ideas and structure questions. If that sounds like something that’s up your alley, join the [Discord](https://alt-f4.blog/discord) to get started!
```

# April 1st Mod Feature

(v 1.2)

## Working title: Funny Factorio Fan Formulations

On the special occasion of Friday April 1st, we are proud to present some humorous picks from Factorio's vast modding scene. We will start with mods that only change visuals or audio to add some unique flavors to your eningeering experiences. Afterwards, we look at some crazy new weapons that spice up combat. Finally, we feature a mod with a level of transportation pizzaz that Willy Wonka himself would be proud of.

### Sights and Sounds

The mod portal features quite a few mods small and large that offer changes to visuals and/or audio. Here is a fun selection:

> **Emotes and Speech Bubbles** 

Reviewer 2: I think we can do the headings like we did in the Modding Behind The Scenes post, where we have the mod author in the header. https://alt-f4.blog/ALTF4-55/

by [Kiplacon](https://mods.factorio.com/user/Kiplacon)

> [This handy mod](https://mods.factorio.com/mod/EmotesAndSpeechBubbles) makes multiplayer interactions a lot livelier, especially when you can't use a mic. Your text in chat also appears over your head. This is already more immersive but the best part is the emotes it provides. The emote menu is opened by clicking the middle mouse button (and customized by right-clicking them). The mod helps enhance your fun and funny moments: There are emotes for partying, coordinating, and of course pressing "F" or declaring "rekt!" when your friend gets hit by a train. 

Reviewer 2: I think here you could expand slightly that it's easier to display emotions, make jokes, or even make fun of your friends, or something like that :)

(Emote gif: easy)

**(Multiplayer image: hard)**

**Disco Science**

by [danielbrauer](https://mods.factorio.com/user/danielbrauer)

> Science is the heart of progress, but [this little mod](https://mods.factorio.com/mod/DiscoScience) also makes it the life of the party! It makes research labs light up with the colors of whichever science packs they are consuming. Excellent for sprucing up your [AFK Zone Discos](https://www.reddit.com/r/factorio/comments/sl4z1v/time_honored_tradition_when_a_player_is_afk_you/)! If you want to go further and add faint music to your disco domes, there is [this other mod](https://mods.factorio.com/mod/LabDansen/metrics).

Reviewer 2: Love the reddit reference!

Also, if you could capture the video clip with the LabDansen sounds then it would be so cool! 

BTW, we prefer videos rather than GIFs. It's much smaller. And don't worry if the way you capture it is too big, we can compress it for you :)

(disco gif: easy)

**Shrek**

by [jerzygorski](https://mods.factorio.com/user/jerzygorski)

He's back and he means business! [This mod](https://mods.factorio.com/mod/shrek) retextures your character as Shrek, and proves that he can do rocket science just like how he can rescue the princess! He mines with his fists and has his own armor and weapon texture too, perfect for getting those pesky bugs out of your swamp planet!

(mining gif: easy) (armor + shooting gif: easy)

**Bumble Bots**

by [Maxreader](https://mods.factorio.com/user/Maxreader)

> If you ever thought about your bots doing everything for you as your little worker bees, [here is a mod](https://mods.factorio.com/mod/bumble-bots) that makes it literal! The mod retextures and resounds combat robots, construction and logistic robots, roboports and logistic chests. The organic structures (especially flowers) among the machines makes an interesing mix.

Reviewer 2: ```suggestion
If you ever thought about your bots doing everything for you as your little worker bees, [here is a mod](https://mods.factorio.com/mod/bumble-bots) that makes it literal! The mod retextures and resounds combat robots, construction and logistic robots, roboports and logistic chests. The organic structures (especially flowers) among the machines makes an interesing environment for your busy bees to buzz.
```

> (logisctic gif: easy) 

Reviewer 2: For this video clip I think we can shamefully cut the first 5 seconds from this video we alread have:
https://alt-f4.blog/ALTF4-35/#mod-spotlight-hall-of-fame-conor_

(gif from mod / image from mod page)

**1000 Sounds to die for**

by [vralfy](https://mods.factorio.com/user/vralfy)

[This sound mod](https://mods.factorio.com/mod/1000sounds) allows you to customize a number of sounds, in particular character deaths. Yes, death in Factorio is usually a tragedy, but you can make it more comedic! Got hit by a train? Hear it honk! Killed by another player? Why not set that to the classic [Wilhelm Scream](https://www.youtube.com/watch?v=r6JK-gRELI0&ab_channel=GamingSoundFX)? You can also customize sounds for a few other events like starting/completing a research, launching a rocket, or players entering/exiting a game. The sound library of this mod borrows from Doom and Quake among some other places. If you like this mod, along the same lines is one for [pain sounds](https://mods.factorio.com/mod/pain), from Doom or custom sources.

(GUI image: easy)

**Choleric cars**

by [Therenas](https://mods.factorio.com/user/Therenas)

> Did your car/tank just crash into rock? Yeah it sucks. [This sound mod](https://mods.factorio.com/mod/cholericcars) lets the engineer express his opinion about it very clearly (sometimes with expletives). It definitely captures the feeling of the moment, often comedically well!

Reviewer 2: ```suggestion
Did your car/tank just crash into rock? Yeah it sucks. [This sound mod](https://mods.factorio.com/mod/cholericcars) lets the engineer express his opinion about it very clearly (sometimes with expletives). It definitely captures the feeling of the moment, often comedically well! And we promise this mod isn't showcased here because the author is our Editor-In-Chief. We simply assume it's his stress outlet for dealing with the rest of us.
```

**(image: hard)**

**More sound mods**

> There are many more little sound mods out there that are specific references. For example, Command & Conquer fans might like this pack of [Red Alert sounds](https://mods.factorio.com/mod/Command_and_Conquer_RedAlert_Sounds) for programmable speakers and viewers of Trupen might enjoy a custom [rock mining](https://mods.factorio.com/mod/TrupenRocks) sound.

Reviewer 2: I think this paragraph is a little lonely. Might work better to work it into the introduction paragraph :)

Author: I can also add more mods: Thomas the tank engine / rick roll

Author: (I did that)

### Crazy Combat

**MLG FactOreo**

by [Kiplacon](https://mods.factorio.com/user/Kiplacon)

> If combat in Factorio is too bland for you these days and you want to summon the essence of Major League Gaming montages, with all their dubstep-rich glory and dank memes (and some expletives) look no further. [This mod delivers](https://mods.factorio.com/mod/MLG_FactOreo). You can craft a sniper rifle for your no-scope pwnage and also craft meme magazines to deal some aptly named "dank damage" with any bullet-firing weapon, including turrets and cars. 

Reviewer 2: Heh, I think this might be fun.
```suggestion
If combat in Factorio is too bland for you these days and you want to summon the essence of Major League Gaming montages, with all their dubstep-rich glory and dank memes (and some expletives) look no further. [This mod delivers](https://mods.factorio.com/mod/MLG_FactOreo). You can craft a sniper rifle for your 360 no-scopepwnage on them noobs. And don't forget to craft meme magazines to deal some aptly named "[dank](https://youtu.be/dQw4w9WgXcQ) damage" with any bullet-firing weapon, including turrets and cars. 
```

Author: The pwnage isnt 360 in the mod sadly. Also cool new link :P

Author: Ok so you can do the 360 manually, which still makes it a bit of a skill

**(1 good combat gif: hard)**

**Train Gun**

by [Twu.4.25](https://mods.factorio.com/user/Twu.4.25)

> Alright, perhaps you are more interested in a weapon that is more... Factorio-style. We all know that the ultimate killer in the game is no mere biter, but the mighty train! Armed with this knowledge, [this mod](https://mods.factorio.com/mod/traingun) delivers the crazy lategame superweapon that is the train gun. Unlocked after nuclear missiles, this powerful gun launches nuclear-fueled locomotives that furiously spin and tear through hoardes of enemies and rocks alike, before exploding magnificently! 

Reviewer 2: ```suggestion
Alright, perhaps you are more interested in a weapon that is more... Factorio-style. We all know that the ultimate killer in the game is no mere biter, but the mighty train! Armed with this knowledge, [this mod](https://mods.factorio.com/mod/traingun) delivers the crazy lategame superweapon that is the train gun. Unlocked after nuclear missiles, this powerful gun launches nuclear-fueled locomotives that furiously spin and tear through hoardes of enemies and rocks alike, before exploding magnificently! Choo Choo!
```

**(1 good combat gif: easy)**

### Wonka Level Logistics: Renai Transportation

by [Kiplacon](https://mods.factorio.com/user/Kiplacon)

> And for last on this mod list we have saved the largest and most imaginative. This [amazing mod](https://mods.factorio.com/mod/RenaiTransportation) introduces a whole **world** of new transportation options that can liven up your factory with so much air time.

Reviewer 2: These paragraphs seems a little spaced out. We could maybe turn it into a bulleted list? What do you think? And we don't need to supply gifs for everything. We need to leave something for the readers imagination, so I think maybe you choose one or two clips you like and we use that? Even the clip on the modpage is amazing. Up to you though :)

Author: Maybe yeah, especially in case the author will want to submit a full piece on the mod late on

Author: Now that we have a neat video by the author for each feature, I think it is best to keep them all.

> And for last on this mod list we have saved the largest and most imaginative. This [amazing mod](https://mods.factorio.com/mod/RenaiTransportation) introduces a whole **world** of new transportation options that can liven up your factory with so much air time.

Reviewer 2: You could also add something like: "Even though this mod is chaotic, it can still be used for beautiful bases. Have a look what the famed creator of [Josef](https://alt-f4.blog/ALTF4-39/) came up with. Something he dubbed 'The Dolphin base'"

It's always fun to shoutout players :)

Reviewer 3: I also saw this iron smelting setup using throwers someone posted on youtube https://www.youtube.com/watch?v=vZOnYDQOXKU

*<Video: disco science>  (not done)*

**Shrek**

by [jerzygorski](https://mods.factorio.com/user/jerzygorski)

He's back and he means business! [This mod](https://mods.factorio.com/mod/shrek) retextures your character as Shrek, and proves that he can do rocket science just like how he can rescue the princess! He mines with his fists and has his own armor and weapon texture too, perfect for getting those pesky bugs out of your swamp planet!

*<video: shrek mining (not done> (not done)*

*<video: shrek in armor + shooting> (not done)*

**Bumble Bots**

by [Maxreader](https://mods.factorio.com/user/Maxreader)

If you ever thought about your bots doing everything for you as your little worker bees, [here is a mod](https://mods.factorio.com/mod/bumble-bots) that makes it literal! The mod retextures and resounds combat robots, construction and logistic robots, roboports and logistic chests. The organic structures (especially flowers) among the machines makes an interesing mix.

*<video: logistic bees> (we can reuse part of an older video)*

*<video: defender wasps> (not done, maybe skip)*

> **Sushi Mod**

Reviewer 3: This reminds me of another mod someone made where they replaced all the items and icons with emojis
https://mods.factorio.com/mod/Emojitorio
